### ISSUE TITLE
SSH Kitten: Cannot find terminfo file since kitty 0.26.5

### ISSUE THREAD

**Issue by aleuffre** [kovidgoyal/kitty#5835]
**Describe the bug**
Hello,

thank you for your hard work on this incredible software :pray: 
since updating to 0.26.5 earlier today, all ssh connections using the ssh kitten fail with error `FileNotFoundError: [Errno 2] No such file or directory: '/usr/lib/kitty/terminfo/kitty.terminfo'`. 
(They work fine without the kitten)

<details>

<summary>Error log</summary>

```
kitty +kitten ssh user@server  -i ~/.ssh/my_key

Connecting...
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/usr/bin/../lib/kitty/__main__.py", line 7, in <module>
    main()
  File "/usr/bin/../lib/kitty/kitty/entry_points.py", line 201, in main
    namespaced(['+', first_arg[1:]] + sys.argv[2:])
  File "/usr/bin/../lib/kitty/kitty/entry_points.py", line 152, in namespaced
    func(args[1:])
  File "/usr/bin/../lib/kitty/kitty/entry_points.py", line 133, in run_kitten
    rk(kitten)
  File "/usr/bin/../lib/kitty/kittens/runner.py", line 112, in run_kitten
    runpy.run_module(f'kittens.{kitten}.main', run_name=run_name)
  File "/usr/lib/python3.10/runpy.py", line 227, in run_module
    return _run_code(code, {}, init_globals, run_name, mod_spec)
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/usr/bin/../lib/kitty/kittens/ssh/main.py", line 756, in <module>
    main(sys.argv)
  File "/usr/bin/../lib/kitty/kittens/ssh/main.py", line 749, in main
    run_ssh(ssh_args, server_args, found_extra_args)
  File "/usr/bin/../lib/kitty/kittens/ssh/main.py", line 714, in run_ssh
    rcmd, replacements, shm_name = get_remote_command(
  File "/usr/bin/../lib/kitty/kittens/ssh/main.py", line 513, in get_remote_command
    sh_script, replacements, shm_name = bootstrap_script(
  File "/usr/bin/../lib/kitty/kittens/ssh/main.py", line 288, in bootstrap_script
    tfd = standard_b64encode(make_tarfile(ssh_opts, dict(os.environ), 'gz' if script_type == 'sh' else 'bz2', literal_env=literal_env)).decode('ascii')
  File "/usr/bin/../lib/kitty/kittens/ssh/main.py", line 196, in make_tarfile
    tf.add(f'{terminfo_dir}/kitty.terminfo', arcname='home/.terminfo/kitty.terminfo', filter=normalize_tarinfo)
  File "/usr/lib/python3.10/tarfile.py", line 1986, in add
    tarinfo = self.gettarinfo(name, arcname)
  File "/usr/lib/python3.10/tarfile.py", line 1865, in gettarinfo
    statres = os.lstat(name)
FileNotFoundError: [Errno 2] No such file or directory: '/usr/lib/kitty/terminfo/kitty.terminfo'
```

</details>

<details>

<summary>**Environment details**</summary>

```
kitty 0.26.5 created by Kovid Goyal
Linux <name> 6.0.0-6-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.0.12-1 (2022-12-09) x86_64
Debian GNU/Linux bookworm/sid pycat /dev/tty

Running under: Wayland
Frozen: False
Paths:
  kitty: /usr/bin/kitty
  base dir: /usr/lib/kitty
  extensions dir: /usr/lib/kitty/kitty
  system shell: /usr/bin/zsh
Loaded config files:
  /etc/xdg/kitty/kitty.conf
  /home/<name>/.config/kitty/kitty.conf

Config options different from defaults:
cursor_text_color       None
update_check_interval   0.0
Colors:
	active_tab_background   #f8f8f2   
	active_tab_foreground   #282a36   
	background              #282a36   
	color0                  #21222c   
	color1                  #ff5555   
	color10                 #69ff94   
	color11                 #ffffa5   
	color12                 #d6acff   
	color13                 #ff92df   
	color14                 #a4ffff   
	color2                  #50fa7b   
	color3                  #f1fa8c   
	color4                  #bd93f9   
	color5                  #ff79c6   
	color6                  #8be9fd   
	color7                  #f8f8f2   
	color8                  #6272a4   
	color9                  #ff6e6e   
	cursor                  #f8f8f2   
	foreground              #f8f8f2   
	inactive_tab_background #6272a4   
	inactive_tab_foreground #282a36   
	mark1_background        #ff5555   
	mark1_foreground        #282a36   
	selection_background    #44475a   
	selection_foreground    #ffffff   
	url_color               #8be9fd   

Important environment variables seen by the kitty process:
	PATH                                /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
	LANG                                en_GB.UTF-8
	SHELL                               /usr/bin/zsh
	DISPLAY                             :0
	WAYLAND_DISPLAY                     wayland-0
	USER                                uffre
	XDG_RUNTIME_DIR                     /run/user/1000
	XDG_CURRENT_DESKTOP                 GNOME
	XDG_MENU_PREFIX                     gnome-
	XDG_SESSION_CLASS                   user
	XDG_SESSION_DESKTOP                 gnome
	XDG_SESSION_TYPE                    wayland

```

</details>

**Additional context**
Problem persists using  `kitty --config NONE`.

**Comment by aleuffre** [comment]
It works on the official installation. Must be some interaction between the debian apt package and the new rules about terminfo https://github.com/kovidgoyal/kitty/commit/1747bbbbcb0090523107836c6d879b4f37316926 I'd imagine.

Either way, thank you very much @page-down !

**Comment by pianohacker** [comment]
Reported downstream at https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1027777 .

**Comment by EvanCarroll** [comment]
Just for reference it's reported to be fixed in Version: 0.26.5-3 which hasn't landed yet in [bookworm](https://packages.debian.org/bookworm/kitty). What you can do until that point is download the package in [sid](https://packages.debian.org/sid/kitty) which on [[kitty_0.26.5.orig.tar.xz]](http://deb.debian.org/debian/pool/main/k/kitty/kitty_0.26.5.orig.tar.xz) in a directory, then run


```sh
xz -d kitty_0.26.5.orig.tar.xz
tar xvvf kitty_0.26.5.orig.tar
sudo cp -R kitty-0.26.5/terminfo /usr/lib/kitty/
sudo chown root:root -R /usr/lib/kitty/terminfo
```

And you're good.
